lib/modules/datasource/docker/index.ts

```diff
@@ -558,6 +594,68 @@ export class DockerDatasource extends Datasource {
     return null;
   }
 
+  @cache({
+    namespace: 'datasource-docker-architecture',
+    key: (
+      registryHost: string,
+      dockerRepository: string,
+      currentDigest: string
+    ) => `${registryHost}:${dockerRepository}@${currentDigest}`,
+    ttlMinutes: 1440 * 28,
+  })
+  public async getImageArchitecture(
+    registryHost: string,
+    dockerRepository: string,
+    currentDigest: string
+  ): Promise<string | null | undefined> {
+    try {
+      const manifestResponse = await this.getManifestResponse(
+        registryHost,
+        dockerRepository,
+        currentDigest,
+        'head'
+      );
+
+      if (
+        manifestResponse?.headers['content-type'] !== MediaType.manifestV2 &&
+        manifestResponse?.headers['content-type'] !== MediaType.ociManifestV1
+      ) {
+        return null;
+      }
+
+      const configDigest = await this.getConfigDigest(
+        registryHost,
+        dockerRepository,
+        currentDigest
+      );
+      if (!configDigest) {
+        return null;
+      }
+
+      const configResponse = await this.getImageConfig(
+        registryHost,
+        dockerRepository,
+        configDigest
+      );
+      if (configResponse) {
+        const architecture = configResponse.body.architecture ?? null;
+        logger.debug(
```

Comment:
```suggestion
        logger.trace(
```

same here

Reply:
Why? This is the only point where it's explicitly logged that the currently specified digest is not a manifest digest but relates to a concrete arch and if yes, which one. Therefore, I don't think this should be hidden behind `trace`.